apps/docs/docs/user/core-concepts.md

```diff
@@ -69,18 +69,22 @@ block GasReserveHttpExtractor oftype HttpExtractor {
 } 
 ```
 
+In the example above, the `url` property of type `text` is defined by the corresponding `HttpExtractor` block type.
 
 ## ValueTypes
 
 A `ValueType` is the definition of a data type of the processed data. Some `Blocks` use `ValueTypes` to define logic (like filtering or assessing the data type in a data sink). We differentiate the following types of `ValueTypes`:
 - `Built-in ValueTypes` come with the basic version of Jayvee. Currently `text`, `decimal`, `integer`, and `boolean` are supported.
-- `Primitive ValueTypes` can be defined by the user to model domain-specific data types and represent a single value. `Constraints` can be added to the `Primitive ValueType` (see below).
+- `Primitive ValueTypes` can be defined by the user to model domain-specific data types and represent a single value.
+  `Constraints` can be added to a `Primitive ValueType` (see [below](#constraints)).
 - `Compound ValueTypes`: UPCOMING.
 
 
 ### Constraints
 
-`Constraints` of `ValueTypes` declare the validity criteria that each concrete value is checked against. The syntax of `Constraints` is similar to the syntax of `Blocks`. The availability of property keys and their respective `ValueTypes` is determined by the type of the `Constraint` - indicated by the identifier after the keyword `oftype`:
+`Constraints` for `ValueTypes` declare the validity criteria that each concrete value is checked against. The syntax 
+of `Constraints` is similar to the syntax of `Blocks`. The availability of property keys and their respective 
+`ValueTypes` is determined by the type of the `Constraint` - indicated by the identifier after the keyword `oftype`:
```

Comment:
These line breaks seem off to me. Maybe make a new line after each sentence instead? I think that is generally a good practice for markdown docs.

Reply:
I agree, will fix this.